packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -771,8 +771,22 @@ class PageTransitionsTheme with Diagnosticable {
       platform = TargetPlatform.iOS;
     }
 
-    final PageTransitionsBuilder matchingBuilder =
-      builders[platform] ?? const ZoomPageTransitionsBuilder();
+    PageTransitionsBuilder getTransitionBuilder() {
+      late PageTransitionsBuilder defaultBuilder;
```

Comment:
late -> final. I think dart is smart enough here to figure out that the variable is never accessed before assignment.